src/app/(admin)/services/providers/staff/page.tsx

```
@@ -0,0 +1,232 @@
+                });
+            } else if (modalMode === 'edit' && selectedStaff) {
+                res = await fetch(`/api/providers/${staffProviderId}/staff/${selectedStaff.id}`, {
+                    method: 'PATCH',
+                    headers: { 'Content-Type': 'application/json' },
+                    body: JSON.stringify(data),
+                });
+            }
+            if (res && !res.ok) {
+                let errorData: { error?: string } = {};
+                const contentType = res.headers.get('content-type');
+                if (contentType && contentType.includes('application/json')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)